.travis.yml

```diff
@@ -4,4 +4,5 @@ go:
   - "1.10.x"
   - master
 
-
```

Comment:
I kept forgetting to format my source code before opening the pull requests. This travis CI addition checks that everything is formatted properly. I can turn this off if you disagree with the philosophy.

Reply:
I'm not sure if it'll cause superfluous failures with different go versions, but if it does we can take it out later.

You overrode the default script section though, so it's not running `go test -v ./...` anymore unless you explicitly add it back.